Add graph, include to release-notes (#2027)
ballerina-platform/ballerina-release: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md

#### New features

##### CLI

> Added new CLI command `bal graph` which resolves the dependencies of the current package and prints the dependency graph in the console. This produces the textual representation of the dependency graph using the DOT graph description language.

Reviewer: ```suggestion
Added a new `bal graph` CLI command, which resolves the dependencies of the current package and prints the dependency graph in the console. This produces the textual representation of the dependency graph using the DOT graph description language.
```

> Introduced `bal graph` CLI command, which resolves the dependencies of the current package and prints the dependency graph in the console. This produces the textual representation of the dependency graph using the DOT graph description language.

Reviewer: ```suggestion
Introduced the `bal graph` CLI command, which resolves the dependencies of the current package and prints the dependency graph in the console. This produces the textual representation of the dependency graph using the DOT graph description language.
```

> Added a new `bal graph` CLI command, which resolves the dependencies of the current package and prints the dependency graph in the console. This produces the textual representation of the dependency graph using the DOT graph description language.

Reviewer 2: ```suggestion
Introduced `bal graph` CLI command, which resolves the dependencies of the current package and prints the dependency graph in the console. This produces the textual representation of the dependency graph using the DOT graph description language.
```

> ```

Reviewer: ```suggestion
```ballerina
```

- [update tool](https://github.com/ballerina-platform/ballerina-update-tool/issues?q=is%3Aissue+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed+label%3AType%2FBug)
- [OpenAPI](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+label%3AType%2FBug+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed)

### Ballerina packages updates

#### New features

> Introduced `include` field under the `[package]` table in `Ballerina.toml`. It accepts a string array of paths to any additional files and directories, which need to be packed in the BALA file. The path should be relative to the package root directory.

Reviewer: ```suggestion
Introduced an `include` field under the `[package]` table in `Ballerina.toml`. It accepts a string array of paths to any additional files and directories, which need to be packed in the BALA file. The path should be relative to the package root directory.
```
Was it intentional to have `package` written within square brackets?

- [update tool](https://github.com/ballerina-platform/ballerina-update-tool/issues?q=is%3Aissue+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed+label%3AType%2FBug)
- [OpenAPI](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+label%3AType%2FBug+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed)

### Ballerina packages updates

#### New features

> Added `include` field under the `[package]` table. It allows providing paths to any additional resources, which need to be packed in the BALA file.

Reviewer: ```suggestion
Added an `include` field under the `[package]` table. It allows providing paths to any additional resources, which need to be packed in the BALA file.
```

> Added an `include` field under the `[package]` table. It allows providing paths to any additional resources, which need to be packed in the BALA file.

Reviewer 2: ```suggestion
Introduced `include` field under the `[package]` table in `Ballerina.toml`. It allows providing paths to any additional resources, which need to be packed in the BALA file.
```

> ```

Reviewer: ```suggestion
```ballerina
```

[package]
org = "samjs"
name = "winery"
version = "0.1.0"
include = [documents/”, "images/sample.png"]
```

> **Info:** The `include` field accepts a string array, which contains the directory or file paths to include in the BALA. The paths should be relative to the package root directory.

Reviewer 2: Shall we have this part in the description as suggested in the comment https://github.com/ballerina-platform/ballerina-release/pull/2027/files#r913165929? @praneesha WDYT?

Author: Removed **Info** and added that content according to the comment https://github.com/ballerina-platform/ballerina-release/pull/2027/files#r913165929.
